apps/browser/src/background/nativeMessaging.background.ts

```diff
@@ -57,26 +55,29 @@ type ReceiveMessageOuter = {
   messageId?: number;
 
   // Should only have one of these.
-  message?: EncString;
+  message?: ReceiveMessage | EncString;
   sharedSecret?: string;
 };
 
 type Callback = {
-  resolver: any;
-  rejecter: any;
+  resolver: (value?: unknown) => void;
+  rejecter: (reason?: any) => void;
+};
+
+type SecureChannel = {
+  privateKey: Uint8Array;
+  publicKey: Uint8Array;
+  sharedSecret?: SymmetricCryptoKey;
+  setupResolve: (value?: unknown) => void;
 };
 
 export class NativeMessagingBackground {
   connected = false;
-  private connecting: boolean;
-  private port: browser.runtime.Port | chrome.runtime.Port;
+  private connecting: boolean = false;
+  private port?: browser.runtime.Port | chrome.runtime.Port;
+  private appId?: string;
 
-  private privateKey: Uint8Array = null;
-  private publicKey: Uint8Array = null;
-  private secureSetupResolve: any = null;
-  private sharedSecret: SymmetricCryptoKey;
-  private appId: string;
-  private validatingFingerprint: boolean;
+  private secure?: SecureChannel;
```

Comment:
🎨 suggestion for future readability:
```suggestion
  private secureChannel?: SecureChannel;
```

Reply:
Fixed in 6d0f0066